hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/event/EntityModifiedEventPayloadIdentifier.java

```diff
@@ -23,15 +22,15 @@
     private final Class<? extends ProxyIdentifiableEntity> parentType;
     private final Class<? extends ProxyIdentifiableEntity> entityType;
     private final EntityModifiedEventType modifiedEventType;
-    private final String eventTypeMessageKey;
+    private final EventNotificationType notificationType;
```

Comment:
Could you please adapt the comment under `#hashCode` and `#equals` that we ignore notificationType

Reply:
done